.github/workflows/npm-publish.yml

```diff
@@ -14,6 +14,7 @@ jobs:
           node-version: '16.x'
           registry-url: 'https://registry.npmjs.org'
       - run: npm install
-      - run: npm publish --workspaces
+      - run: npm run build
```

Comment:
Why is building necessary here when it wasn't previously? Is it because the `dist` folder needs to be in the publish?

Reply:
Yes. This is related to Preston's change for user agent we now need to build in the publish workflow so that the GITHUB_SHA is captured as host version